libnavui-tripprogress/src/main/res/values/strings.xml

```diff
@@ -0,0 +1,4 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
+    <string name="mapbox_trip_progress_text_divider_landscape" translatable="false"> \u2022</string>
```

Comment:
Is this supposed to have a space?  ...se"> \u2022</st..

Reply:
Yes if I don't add a space it always draw the artifact as seen in the issue